### PR TITLE
Fixes for pydocstyle rule D412

### PR DESCRIPTION
**Pull request by morrme**
Ref. #1170

### CODE REVIEW

File: Bio/Application/__init__.py [hunk 1]
```diff
@@ -539,7 +539,6 @@ class _Option(_AbstractParameter):
     take a value, use the _Switch object instead.
 
     Attributes:
-
```
Comment (peterjc) [inline Bio/Application/__init__.py]: As an aside, all these old-style docstrings using lower-case o as a bullet point need updating at some point, e.g. RST bullet points would be simplest.

**Comment by peterjc** [comment]
Many of the example you've changed need other work to adopt RST markup, so in itself this doesn't directly make the documentation any clearer - but I'm happy about bringing it slightly more in line with PEP257 anyway.

**Comment by peterjc** [comment]
Cherry picked and merge conflict fixed, see 77df7d82516f9b75a6932f5e2b0eb920bb0ea18b

Thank you.

File: Bio/Application/__init__.py (hunk 1 above)
Comment (peterjc) [inline Bio/Application/__init__.py]: e.g. I did some in e8bbaa89b30bd6e55d4ed5a36ab62a17529e76e0

**Comment by morrme** [comment]
@peterjc in accordance with the issue description, I was only focused on one or two rules at a time. I thought that would keep things organized, even if it meant that I'd have to revisit a file later on to make another change.

**Comment by peterjc** [comment]
@morrme good point. I've added this to #1170:

*Update*: You may find some of the modules have loads of issues, and instead it may make sense to work on all the docstring style or reStructuredText problems in that module at once. That's fine too - but again please comment on this issue to avoid duplication of effort.

**Comment by morrme** [comment]
@peterjc Duplication of work is definitely a risk in this approach, as well as an increased number of conflicts. The other challenge with working on multiple problems at once is that there may be some things that are intentional, as you've pointed out, so changing things for one or two related rules at a time allows the reviewer to be able to point these out before too much is overhauled.  I can understand it may be annoying as a reviewer to see one change but then three other things _right there_ that should be fixed, but hopefully once each rule is addressed those issues will all eventually be taken care of.

**Comment by peterjc** [comment]
Eventually :)

Right now its just you and me doing these edits so it should be OK...

**Pull request closed without merging**